ystafdb/config_parser.py

```diff
@@ -1,13 +1,17 @@
+import os
 import json
-
+from pathlib import Path
 
 def get_config_data():
     providers = []
     datasets = []
     providersRequiredFields = set()
     datasetsRequiredFields = set()
 
-    with open('ystafdb/data/config.json') as json_file:
+    if not os.path.exists(Path("config.json")):
+        exit("No config.json file")
```

Comment:
should use `sys.exit` and return a status code > 0 for errors

Reply:
a46b58f reflects this wish